service/api_patch.go

```diff
@@ -166,7 +167,8 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if p.Alias == evergreen.CommitQueueAlias && !patch.IsMailboxDiff(data.PatchString) {
+	data.PatchString = strings.TrimSpace(data.PatchString)
+	if p.Alias == evergreen.CommitQueueAlias && len(data.PatchString) != 0 && !patch.IsMailboxDiff(data.PatchString) {
```

Comment:
I don't think there's a use case for an empty module patch

Reply:
Fair but I'd rather remain consistent